### PR TITLE
AzureRT PR S51 - Updating Start/Stop Emulator related cmdlets to work with latest .NET release

### PR DESCRIPTION
**Pull request by huangpf**
- Updated ServiceDefinition and ServiceConfiguration schema files to unblock deserialization
- Updated Version number validation to relax constraint 
- Updated StorageEmulator invocation due to StorageEmulator executable rename

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@huangpf__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (phuang). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

File: src/ServiceManagement/Services/Commands.Utilities/Common/AzureTools/AzureTool.cs [hunk 1]
```diff
@@ -94,13 +94,11 @@ private static string GetSdkVersionRegistryValue()
...
                     }
-                    version = key.GetSubKeyNames().FirstOrDefault(n => n == AzureSdkVersionInfo.SupportAzureSdkVersionRegKey);
 
-                    if (string.IsNullOrEmpty(version) && key.GetSubKeyNames().Length > 0)
-                    {
-                        throw new InvalidOperationException(string.Format(Resources.AzureSdkVersionNotSupported, AzureSdkVersionInfo.SupportAzureSdkVersionRegKey));
-                    }
-                    else if (string.IsNullOrEmpty(version) && key.GetSubKeyNames().Length == 0)
+                    // select the latest version of SDK installed on the machine.
+                    version = key.GetSubKeyNames().Last();
```
Comment (huangpf) [inline src/ServiceManagement/Services/Commands.Utilities/Common/AzureTools/AzureTool.cs]: It seems, if ```key.GetSubKeyNames()``` is empty, ```Last()``` would throw exception. Would it be a concern?

**Comment by huangpf** [comment]
LGTM.

File: src/ServiceManagement/Services/Commands.Utilities/Common/AzureTools/AzureTool.cs (hunk 1 above)
Comment (waynekuo) [inline src/ServiceManagement/Services/Commands.Utilities/Common/AzureTools/AzureTool.cs]: You're right. Let me change that to LastOrDefault() so it won't throw when an item is not found.

**Pull request merged**